config/default/common/config/metadata/stories/clouds/step003.md

```diff
@@ -0,0 +1 @@
+A less typical image you may have seen of tropical storms is this night time image from the Day/Night Band of the VIIRS instrument on board the joint NASA/NOAA Suomi NPP satellite. It shows the swirling clouds from Hurricanes Katia, Irma and Jose crossing the North Atlantic Ocean, Caribbean Sea and Gulf of Mexico on 8 September 2017.
```

Comment:
I found it a bit tricky to see the rightmost hurricane on this step since it was being blocked by the tour story step.

Reply:
It's a great example of nighttime cloud imagery, though!

Reply:
I updated this and zoomed out a bit.